### ISSUE TITLE
Disabled button styling doesn't meet contrast guidelines

### ISSUE THREAD

**Issue by mschile** [cypress-io/cypress-design#523]
The styling for a [disabled button](https://design.cypress.io/components/vue/Button.html#possible-variants) doesn't meet a11y guidelines for contrast.

![Image](https://github.com/user-attachments/assets/8c5d8a67-a110-4d5c-bc2a-4a66b82fe319)
